### ISSUE TITLE
Failed to download package (rpm)

### ISSUE THREAD

**Issue by mariuszskon** [niklasb/libc-database#41]
```
Getting rpm
  -> Location: http://rpmfind.net/linux/openmandriva/cooker/repository/x86_64/main/release/glibc-2.33-4-omv4040.x86_64.rpm
  -> ID: libc-2.33-4-omv4040.x86_64
  -> Downloading package
  -> Extracting package
  -> Writing libc /tmp/tmp.dyI8IxHGco/lib64/libc.so.6 to db/libc-2.33-4-omv4040.x86_64.so
  -> Writing symbols to db/libc-2.33-4-omv4040.x86_64.symbols
  -> Writing version info
Getting rpm
  -> Location: http://rpmfind.net/linux/fedora/linux/updates/testing/34/Everything/x86_64/Packages/g/glibc-2.33-4.fc34.x86_64.rpm
  -> ID: libc-2.33-4.fc34.x86_64
  -> Downloading package
Failed to download package from http://rpmfind.net/linux/fedora/linux/updates/testing/34/Everything/x86_64/Packages/g/glibc-2.33-4.fc34.x86_64.rpm
The command '/bin/sh -c ~/.libc-database/get all' returned a non-zero code: 1
```

**Comment by mariuszskon** [comment]
I'm not sure if it's a good idea to stop everything if we get a single 404.

**Comment by niklasb** [comment]
I agree with you, and I will gladly accept a PR that changes the corresponding functions to return early instead of exiting

**Comment by mariuszskon** [comment]
@niklasb see #42 (and #43 if you have time)

**Comment by niklasb** [comment]
should be fixed by 460851d300b802f71e9ef1ba4a9f5080ff7f4c37
